kore-rpc-types/src/Kore/JsonRpc/Types/Log.hs

```diff
@@ -46,7 +48,8 @@ data LogRewriteResult
 
 data LogEntry
     = Rewrite
-        { result :: LogRewriteResult
+        { originalTerm :: Maybe KoreJson
```

Comment:
do we need the original term? the intention for the rewrite trace was to be used when debugging proofs with single step turned on in pyk, so we already have the `originalTerm` anyway.

Reply:
Agreed, let's drop it after all.